tests/test_pyglmnet.py

```diff
@@ -331,7 +331,7 @@ def rmse(a, b):
     clf = ElasticNet(alpha=alpha, l1_ratio=l1_ratio, tol=1e-5)
     clf.fit(X, Y)
     glm = GLM(distr='gaussian', alpha=l1_ratio, reg_lambda=alpha,
-              solver=solver, tol=1e-5, max_iter=70)
+              solver=solver, tol=1e-5, max_iter=10000)
```

Comment:
is this going to slow down the test?

Reply:
This test took about 3 seconds in my environment (CPU: `Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz`) for each solver. 
Do you feel this result is slow?  
If you feel so, I will make the test more efficient.

Reply:
It's a bit slow I would say. Our solvers are kind of slow, so that's the reason but if it can be made faster, it would be great! 

Also there is a danger that in the future someone may reduce the number of iterations without realizing what it was testing for, because it is a slow test. Perhaps add a comment if you can't make it faster explaining why `max_iter=10000` is needed.

Reply:
I think that the simplest way to make the test faster is to reduce the dimension of the `X`.  
For example, by changing the `n_features` from `1000` to `500`, this test took only about 0.3 seconds  in the same environment for each solver(and `max_iter` can be set to `500`).  
Do we need to keep `n_features` at `1000`?

Reply:
changing to `n_features=500` sounds fair to me as long as the test still fails on master but passes on the pull request branch.

Reply:
@jasmainak 
OK, I changed 'n_features=500` and some other parameters to fail on master.